src/include/network/routines.rb

```diff
@@ -746,6 +746,17 @@ def has_carrier?(dev_name)
       ).to_i != 0
     end
 
+    def has_physical_port_id?(dev_name)
```

Comment:
not ruby name. in ruby it should be `physical_port_id?` stuff like is or have/s is not needed.